pyproject.toml

```diff
@@ -29,8 +29,6 @@ select = [
     # "UP",
     # flake8-bugbear
     "B",
-    # flake8-simplify
-    "SIM",
```

Comment:
Why is this removed?

Reply:
Because I think it introduces a lot of changes, and it will just make the diff a lot larger

Reply:
Can we keep the rules that were there in the original ruff version?

Reply:
sg